openbrews/recipe/RecipeUtils.js

```diff
@@ -0,0 +1,131 @@
+angular.module('openbrews.recipeUtils', [])
+.service('RecipeUtils', function () {
+
+    // calculates IBU given a recipe and OG (original gravity).
+    this.calculateIbu = function (recipe, og) {
+        var hops = recipe.hops.filter(function (hop) {
+            return hop.aa;
+        });
+
+        var gravity = this.boilGravity(recipe, og);
```

Comment:
Sadly not, I've been reading up on craft brews as I go along. OG means original gravity I believe, which is the amount of sugar in the recipe _before_ brewing begins.
http://www.brewersfriend.com/2009/02/04/beer-styles-original-gravity-and-final-gravity-chart/